src/Common/Menus/Traits/Submenu.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Provides methods and properties for submenus.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Common\Menus
+ */
+
+namespace TEC\Common\Menus\Traits;
+
+trait Submenu {
```

Comment:
The traits are designed to be used when extending the abstract menu class - a la carte menu pages.
See the way they were implemented in the TEC PR.

I can see where it might make more sense to make Submenu the default though 🤔